rollbar-payload/src/main/java/com/rollbar/payload/data/body/Frame.java

```diff
@@ -217,22 +217,30 @@ public Frame keywordArgs(Map<String, Object> keywordArgs) {
 
     public Map<String, Object> asJson() {
         Map<String, Object> obj = new LinkedHashMap<String, Object>();
-        if (filename() != null)
-            obj.put("filename", filename());
-        if (lineNumber() != null)
-            obj.put("lineno", lineNumber());
-        if (columnNumber() != null)
-            obj.put("colno", columnNumber());
-        if (method() != null)
-            obj.put("method", method());
-        if (code() != null)
-            obj.put("code", code());
-        if (context() != null)
-            obj.put("context", context());
-        if (args() != null)
-            obj.put("args", args());
-        if (keywordArgs() != null)
-            obj.put("kwargs", keywordArgs());
+        String filename = filename();
```

Comment:
I think this would be more readable if all of the variables were defined at the top of this method and then checked below. Other than this, LGTM.